lib/process-services/task-list/components/task-filters.component.ts

```diff
@@ -151,22 +131,47 @@ export class TaskFiltersComponent implements OnInit, OnChanges {
         this.appsProcessService.getDeployedApplicationsByName(appName).subscribe(
             application => {
                 this.getFiltersByAppId(application.id);
-                this.selectTaskFilter(this.filterParam, this.filters);
             },
             (err) => {
                 this.error.emit(err);
             });
     }
 
+    /**
+     * Create default filters by appId
+     * @param appId
+     */
+    getDefaultFilters(appId?: number) {
```

Comment:
Can you call it create if the goal of this method creates default filter?

Reply:
Done.